tds-itemselection-impl/src/main/java/tds/itemselection/impl/item/CsetItem.java

```diff
@@ -136,9 +136,15 @@ public CsetGroup getParentGroup () {
   // / Cset1 factory may remove this item from the pool even though the base
   // item is active
   // / </summary>
```

Comment:
If an item group is passed into this adaptive selector algorithm, it's an artificial group and should be considered. Parent status is unimportant in this case.